bedrock/mozorg/templates/mozorg/contribute/tasks/follow-mozilla.html

```diff
@@ -0,0 +1,28 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/tasks/taskview-base.html" %}
+
+{% block page_title %}{{ _('Follow @MozParticipate Twitter Account') }}{% endblock %}
+
+{% block taskview_title %}{{ _('Follow @MozParticipate and see how others are contributing!') }}{% endblock %}
+{% block taskview_lead %}{{ _('There are so many ways to get involved with Mozilla and support the browser. Follow Firefox and learn more about the browser, and how you can pitch in!') }}{% endblock %}
+
+{% block task_content %}
+  <ol class="task-steps single">
+    <li id="step_one" class="one">
+      <header>
+        <h3>{{ _('Follow @MozParticipate') }}</h3>
+      </header>
+      <div class="step-content">
+        <img src="{{ static('/media/img/contribute/signup/startmozilla.png') }}" width="600" height="338" alt="" />
+        <a href="https://twitter.com/intent/follow/?screen_name=mozparticipate" rel="external" class="follow-mozilla" role="button">+ Follow @MozParticipate</a>
```

Comment:
I don't actually believe we need this (not in the design) so, just going to remove them.